### DISCUSSION TITLE
Discussions / mark threads as closed/answered/outdated

### DISCUSSION THREAD

**Discussion by moving-bits** [community/community#4026] [Discussions]
Thanks for implementing the discussions feature. One thing I'm missing is an easy way to distinguish between active threads and those that do no longer require any more attention from us, because they are outdated / closed / questions answered. Those threads should be greyed out, moved to the end or whatever else draws less attention to them / makes their state clearly visible.

You do have a "question is answered" already, which may do the trick (at least partly), but that flag seems not to be available for all types of discussions.

**Comment by Relequestual** [comment]
There are multiple use cases where a Discussion should be able to be marked as concluded.
Rather than providing specific labels, why not allow customisation of the label. It could be "concluded" or "resolved". "answered" may not be semantically correct for the use case.

IMHO the important thing here is to be able to have a filter on by default which hides non-active discussions. What that flag is called could be different for each catagory, and in some cases maybe multiple possible labels or "end points" might be appropriate.

**Comment by Relequestual** [comment]
Close in favour of https://github.com/github/feedback/discussions/3097 ?

**Reply by moving-bits** [reply]
> Close in favour of #3097 ?

If there would be a way to close discussions... ;-)
